### ISSUE TITLE
Add Further css convertion functionallity

### ISSUE THREAD

**Issue by TheRealSyler** [TheRealSyler/sass-formatter#8]
Make this work `.class {margin:  20px; padding: 20px}`, in `src/formatters/format.convert`.
Also Refactor `src/formatters/format.convert`.

Edit: Fixed

**Comment by TheRealSyler** [comment]
fix
``` @for $i from 0 through 3
  .nav-#{map-get($directions, $i)}
    #{map-get($directions, $i)}: 1rem
    ///R
    animation: nav-#{map-get($directions, $i)}-start 200ms ease
    @if $i % 2 == 0
      @include CenterX()
    @else
      @include CenterY()```

**Comment by TheRealSyler** [comment]
fixed in d4576c587fef83b31c270373b144d807f3ec5704
